src/components/LinearProgressBar/LinearProgressBar.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+import {linearProgressBarDriverFactory} from './LinearProgressBar.driver';
+import {LinearProgressBar, LinearProgressBarProps} from './LinearProgressBar';
+import { linearProgressBarTestkitFactory } from '../../testkit';
+import {linearProgressBarTestkitFactory as enzymeLinearProgressBarTestkitFactory} from '../../testkit/enzyme';
+import { runTestkitExistsSuite } from '../../common/testkitTests';
+
+describe('LinearProgressBar', () => {
+
+    const createDriver = createDriverFactory(linearProgressBarDriverFactory);
+
+    const defaultProps = {
+        value: 40
+    }
+   
+    it('should exist', () => {
+      const driver = createDriver(<LinearProgressBar {...defaultProps}/>);
+      expect(driver.exists()).toBe(true);
+    });
+
+    describe('on error', () => {
+
+        const errorProps = {
+            error: true,
+            errorMessage: 'No soup for you',
+            showProgressIndication: true
+        }
+
+        it('should display tooltip text only on hover', () => {
```

Comment:
Missing another test to check if the errorMessage is the text we have inside the tooltip. You can consult Shai which sits right behind you

Reply:
done